src/test/java/org/jitsi/meet/test/ActiveSpeakerTest.java

```diff
@@ -94,37 +97,40 @@ private void assertOneDominantSpeaker(WebDriver driver)
         for (WebElement el : dominantSpeakerIndicators)
         {
             if (el.isDisplayed())
+            {
                 speakers++;
+            }
         }
 
         assertEquals(
-            1, speakers,
+            1,
+            speakers,
             "Wrong number of dominant speaker indicators.");
     }
 
     private void muteAllParticipants()
     {
         MuteTest muteTest = new MuteTest(this);
-        muteTest.muteOwnerAndCheck();
-        muteTest.muteParticipantAndCheck();
-        muteTest.muteThirdParticipantAndCheck();
+        muteTest.muteParticipant1AndCheck();
+        muteTest.muteParticipant2AndCheck();
+        muteTest.muteParticipant3AndCheck();
```

Comment:
🥇

Reply:
I normalized the names to "participant1", "participant2" and "participant3". For example, in different places we used to refer to participant2 as: "participant", "secondParticipant", "peer", "peer2" and "2nd".